features/step_definitions/coronavirus_local_restrictions_steps.rb

```diff
@@ -0,0 +1,12 @@
+When "I visit the postcode checker" do
+  visit_path "/find-coronavirus-local-restrictions"
+end
+
+When "I enter a valid postcode" do
+  fill_in "Enter your postcode", with: "E18QS"
```

Comment:
Are most users going to fill it with with a space?
Assuming that won't matter for the code but should we test for that?

Reply:
I don't think here is the right place for that level of testing. This a very high level "is it still up" test. But we should test for it in collections, if we're not already.